README.md

```diff
@@ -187,75 +143,76 @@ At the root of this repository, execute `setup.sh`. It will:
-Testing endpoint http://[ip-2]:8080
-Endpoint http://[ip-2]:8080 is responding.
+Testing endpoint http://35.230.123.231:8080
+Endpoint http://35.230.123.231:8080 is responding.
+**** http://35.230.123.231:8080/prime/10
 The sum of all primes less than 10 is 17
 The factorial of 10 is 3628800
 
 Validating Kubernetes Webapp...
-Testing endpoint http://[ip-3]
-Endpoint http://[ip-3] is responding.
+Testing endpoint http://35.190.89.136
+Endpoint http://35.190.89.136 is responding.
+**** http://35.190.89.136/prime/10
 The sum of all primes less than 10 is 17
 The factorial of 10 is 3628800
 ```
 
+Of course, the IP addresses will likely differ for your deployment.
+
 ## Load Testing
 
-In a new console window, execute (using ip-3 from above)
+In a new console window, execute the following, replacing `[IP_ADDRESS]` with the IP address and port from your validation output from the previous step. Note that the Kubernetes deployment runs on port `80`, while the other two deployments run on port `8080`:
 ```console
-ab -c 120 -t 60  http://[ip-3]/prime/10000
+ab -c 120 -t 60  http://[IP_ADDRESS]/prime/10000
 ```
-ApacheBench (`ab`) will execute 120 concurrent requests against ip-3 for 1
-minute. This single replica is insufficient to handle this volume of requests.
+ApacheBench (`ab`) will execute 120 concurrent requests against the provided endpoint for 1 minute. The demo application's replica is insufficiently sized to handle this volume of requests.
 
-![screenshot](./images/ab_load-test-1.png)
+This can be confirmed by reviewing the output from the `ab` command. A `Failed requests` value of greater than 0 means that the server couldn't respond successfully to this load:
 
-This can be confirmed by reviewing the output from the `ab` command. A
-"Failed requests" value of more than 0 means that the server couldn't
-respond successfully for this amount of load.
+![screenshot](./images/ab_load-test-1.png)
 
-A good mechanism for scaling this type of service is via horizontal scaling
-(adding additional instances behind a load balancer). For either of the
-first two deployments (Debian or COS), this would take several steps:
+A good mechanism for scaling this type of service is via horizontal scaling (adding additional instances behind a load balancer). For either of the first two deployments (Debian or COS), this would take several steps:
 
 1.  Create a load balancer.
-1.  Create additional instances.
+1.  Create additional application instances.
 1.  Move all instances behind the load balancer.
 
-This is a very involved process and is out of scope for this demonstration.
+This is an involved process and is out of scope for this demonstration.
 
 For the third (Kubernetes) deployment the process is far easier:
 
 ```console
 kubectl scale --replicas 3 deployment/prime-server
 ```
 
-After allowing 30 seconds for the replicas to initialize, re-run.
+After allowing 30 seconds for the replicas to initialize, re-run the load test:
```

Comment:
can we leverage a readiness check or is there something more manual that is needed?